src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiLiveTest.java

```diff
@@ -119,6 +119,18 @@ public void testListPermissionByGroup() {
         assertThat(permissionsPage.values()).isEmpty();
     }
 
+    @Test(dependsOnMethods = "testGetRepository")
+    public void testCreatePermissionByGroupNonExistent() {
+        boolean success = api().createPermissionsByGroup(projectKey, repoKey, "REPO_WRITE", randomString());
```

Comment:
This actually returns a `204 no content`. Can we update the test to reflect as much as well as the mock test to set the correct return code?